runtime/src/serde_snapshot/older.rs

```diff
@@ -0,0 +1,272 @@
+
+#[derive(Default, Clone, PartialEq, Debug, Deserialize, Serialize, AbiExample)]
+struct UnusedAccounts {
+    unused1: HashSet<Pubkey>,
+    unused2: HashSet<Pubkey>,
+    unused3: HashMap<Pubkey, u64>,
+}
+
+// Deserializable version of Bank which need not be serializable,
+// because it's handled by SerializableVersionedBank.
+// So, sync fields with it!
+#[derive(Clone, Deserialize)]
+struct DeserializableVersionedBank {
+    blockhash_queue: BlockhashQueue,
+    ancestors: AncestorsForSerialization,
+    hash: Hash,
+    parent_hash: Hash,
+    parent_slot: Slot,
+    hard_forks: HardForks,
+    transaction_count: u64,
+    tick_height: u64,
+    signature_count: u64,
+    capitalization: u64,
+    max_tick_height: u64,
+    hashes_per_tick: Option<u64>,
+    ticks_per_slot: u64,
+    ns_per_slot: u128,
+    genesis_creation_time: UnixTimestamp,
+    slots_per_year: f64,
+    #[allow(dead_code)]
+    unused: u64,
+    slot: Slot,
+    epoch: Epoch,
+    block_height: u64,
+    collector_id: Pubkey,
+    collector_fees: u64,
+    fee_calculator: FeeCalculator,
+    fee_rate_governor: FeeRateGovernor,
+    collected_rent: u64,
+    rent_collector: RentCollector,
+    epoch_schedule: EpochSchedule,
+    inflation: Inflation,
+    stakes: Stakes,
+    #[allow(dead_code)]
+    unused_accounts: UnusedAccounts,
+    epoch_stakes: HashMap<Epoch, EpochStakes>,
+    is_delta: bool,
+}
+
+impl From<DeserializableVersionedBank> for BankFieldsToDeserialize {
+    fn from(dvb: DeserializableVersionedBank) -> Self {
+        BankFieldsToDeserialize {
+            blockhash_queue: dvb.blockhash_queue,
+            ancestors: dvb.ancestors,
+            hash: dvb.hash,
+            parent_hash: dvb.parent_hash,
+            parent_slot: dvb.parent_slot,
+            hard_forks: dvb.hard_forks,
+            transaction_count: dvb.transaction_count,
+            tick_height: dvb.tick_height,
+            signature_count: dvb.signature_count,
+            capitalization: dvb.capitalization,
+            max_tick_height: dvb.max_tick_height,
+            hashes_per_tick: dvb.hashes_per_tick,
+            ticks_per_slot: dvb.ticks_per_slot,
+            ns_per_slot: dvb.ns_per_slot,
+            genesis_creation_time: dvb.genesis_creation_time,
+            slots_per_year: dvb.slots_per_year,
+            slot: dvb.slot,
+            epoch: dvb.epoch,
+            block_height: dvb.block_height,
+            collector_id: dvb.collector_id,
+            collector_fees: dvb.collector_fees,
+            fee_calculator: dvb.fee_calculator,
+            fee_rate_governor: dvb.fee_rate_governor,
+            collected_rent: dvb.collected_rent,
+            rent_collector: dvb.rent_collector,
+            epoch_schedule: dvb.epoch_schedule,
+            inflation: dvb.inflation,
+            stakes: dvb.stakes,
+            epoch_stakes: dvb.epoch_stakes,
+            is_delta: dvb.is_delta,
+            prior_roots: Vec::<Slot>::default(),
```

Comment:
@brooksprumo : Here's the `from` implementation of the "older" where the `prior_roots` is set to `Vec::<Slot>::default()`.